commands/bench-bm/bench-vm.sh

```diff
@@ -3,5 +3,5 @@
 set -eu -o pipefail
 shopt -s inherit_errexit
 
-# this is a fallback for `bench` command running on BM3 machine
+# this is a fallback for `bench` command running on VM machine
```

Comment:
This comment doesn't need to be changed, `bench-bm` is a fallback to run `bench` on bm3

Reply:
I guess we just need to remove whole bench-vm

Reply:
Cumulus and rococo are still on bm weights because of failed benches. The idea was to make a switcharoo and set vm runners as a default. BTW @mordamax how to update bot help to this changes?

Reply:
Got it, makes sense, but remove files in folder accordingly to folder too

bench-vm.cmd.json are the sources of `bot help`, so as long as you rename files and content there, that will build the help page accordingly